src/vs/workbench/parts/debug/electron-browser/debugViews.ts

```diff
@@ -60,7 +60,7 @@ export class VariablesView extends CollapsibleView {
 		@IListService private listService: IListService,
 		@IThemeService private themeService: IThemeService
 	) {
-		super({ ...options, sizing: ViewSizing.Flexible, ariaHeaderLabel: nls.localize('variablesSection', "Variables Section") }, keybindingService, contextMenuService);
+		super({ ...(options as IViewOptions), sizing: ViewSizing.Flexible, ariaHeaderLabel: nls.localize('variablesSection', "Variables Section") }, keybindingService, contextMenuService);
```

Comment:
These casts are necessary to prevent passing the  extra properties unique to `IViewletViewOptions` in to the parameter type of `ICollapsibleViewOptions`